### DISCUSSION TITLE
Extra subtitle track for timestamp in exported videos

### DISCUSSION THREAD

**Discussion by cckit** [blakeblackshear/frigate#11355] [Ideas]
The downloaded Nest camera videos has an extra subtitle track for timestamp.
I think it is pretty nice since timestamp could be hidden and video re-encoding is probably not required.
Can it be done with FFmpeg args? Thanks!

How the subtitle looks like in QuickTime:
<img width="466" alt="Subtitle" src="https://github.com/blakeblackshear/frigate/assets/4402060/34869e2f-b6d6-4e72-a947-c25744d4fd28">

3rd track for subtitle:
<img width="344" alt="Track info" src="https://github.com/blakeblackshear/frigate/assets/4402060/6127a246-d033-47b4-be75-c6ab598cbfd3">

**Comment by hawkeye217** [comment]
Looks like you can use the subtitles filter:

https://trac.ffmpeg.org/wiki/HowToBurnSubtitlesIntoVideo

You may need to reencode, though.
